controllers/chat/group/removeGroupMember.js

```diff
@@ -16,6 +24,28 @@ const removeGroupMember = async (req, res) => {
       const {newChannelName, betterChannelMember, betterChannelMemberObject, updatedChannel} =
         await BetterSocialCore.chat.updateBetterChannelMembers(channel, channelApiResponse, true);
 
+      const client = new StreamChat(
+        Environment.GETSTREAM_API_KEY,
+        Environment.GETSTREAM_API_SECRET
+      );
+      const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);
+
+      const textOwnUser = `You removed ${targetUserModel.username} from this group`;
+      const textTargetUser = `You has been removed from this group`;
+      const textDefaultUser = `${ownUser.username} removed ${targetUserModel.username} from this group`;
+      const members = betterChannelMember.map((member) => member.user_id);
+
+      await currentChannel.sendMessage({
+        text: textDefaultUser,
+        own_text: textOwnUser,
+        other_text: textTargetUser,
+        type: 'system',
+        user_id: userId,
+        system_user: userId,
+        isSystem: true,
+        members: members
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The instantiation of the `StreamChat` client should ideally be done once and reused throughout your application, rather than being created anew in each function. This can improve performance by reducing unnecessary object creation and garbage collection. Consider moving the instantiation to a shared module or service.

Also, error handling is missing for the `sendMessage` method. If an error occurs while sending the message, it will not be caught and handled properly, which could lead to unexpected behavior. 

Here's how you might refactor this:

```diff
-      const client = new StreamChat(
-        Environment.GETSTREAM_API_KEY,
-        Environment.GETSTREAM_API_SECRET
-      );
+      const client = require('../../../services/streamChatClient');

       const currentChannel = await client.channel(CHANNEL_TYPE_STRING.GROUP, channelId);

       const textOwnUser = `You removed ${targetUserModel.username} from this group`;
       const textTargetUser = `You has been removed from this group`;
       const textDefaultUser = `${ownUser.username} removed ${targetUserModel.username} from this group`;
       const members = betterChannelMember.map((member) => member.user_id);

-      await currentChannel.sendMessage({
+      try {
+        await currentChannel.sendMessage({
           text: textDefaultUser,
           own_text: textOwnUser,
           other_text: textTargetUser,
           type: 'system',
           user_id: userId,
           system_user: userId,
           isSystem: true,
           members: members
-      });
+        });
+      } catch (error) {
+        console.error(`Failed to send system message: ${error}`);
+        // Handle or throw the error as appropriate for your application
+      }
```

In the above code, I've assumed that you have a `streamChatClient.js` file in your `services` directory that exports a pre-configured instance of `StreamChat`. If not, you'll need to create one. This will allow you to reuse the same client across your application.

Also, I've added a try-catch block around the `sendMessage` call to handle any errors that might occur when sending the message. You should replace the `console.error` and subsequent comment with appropriate error handling for your application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->